### PR TITLE
adding OscillatorNode

### PR DESCRIPTION
**Pull request by chrisdavidmills**

### CODE REVIEW

**Comment by chrisdavidmills** [comment]
Done

File: api/OscillatorNode.json [hunk 1]
```diff
@@ -0,0 +1,467 @@
...
+              "version_added": "14"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "25"
```
Comment (Elchi3) [inline api/OscillatorNode.json]: I think this needs a note:
`notes: "Before Firefox 30, the <code>when</code> parameter is not optional."`

File: api/OscillatorNode.json [hunk 2]
```diff
@@ -0,0 +1,467 @@
...
+            },
+            "firefox_android": {
+              "version_added": "26"
```
Comment (Elchi3) [inline api/OscillatorNode.json]: ditto

File: api/OscillatorNode.json [hunk 3]
```diff
@@ -0,0 +1,467 @@
...
+              "version_added": "14"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "25"
```
Comment (Elchi3) [inline api/OscillatorNode.json]: Same note here.

File: api/OscillatorNode.json [hunk 4]
```diff
@@ -0,0 +1,467 @@
...
+            },
+            "firefox_android": {
+              "version_added": "26"
```
Comment (Elchi3) [inline api/OscillatorNode.json]: ditto

**Comment by chrisdavidmills** [comment]
Done.

**Pull request merged**